doc/rtmpwebhookauth.md

```diff
@@ -1,24 +1,28 @@
 # RTMP Webhook Authentication
 
-Incoming RTMP streams can be authenticated using Webhooks. To use it you need to implement own web server which will be called by Livepeer node when new RTMP stream received to check if that stream is allowed.
+Incoming RTMP streams can be authenticated using Webhooks. To provide these webhooks, node operators must implement their own web service / endpoint to be accessed only by the Livepeer node. As new RTMP streams appear, the Livepeer node will hit this endpoint to determine whether the given stream is allowed.
```

Comment:
> To provide these webhooks,

"To use these webhooks," ?

> the Livepeer node will hit this endpoint

"the Livepeer node will call this endpoint"

Reply:
thanks.  addressed in cbe12e2e45fe29193b9385c06451e9af5b0176c7